types/connector_lock.yaml

```diff
@@ -0,0 +1,37 @@
+description: EV and EVSE board support types
+types:
+  ConnectorLockError:
+    description: >-
+      Errors for Connector Lock
+
+      These following error types give more details about connector lock errors.
```

Comment:
Probably either "These errors" or "The following errors"

Reply:
fixed